doc/api/assert.md

```diff
@@ -121,9 +124,10 @@ changes:
 
 Generally identical to `assert.deepEqual()` with three exceptions:
 
-1. Primitive values are compared using the [Strict Equality Comparison][]
-  ( `===` ). Set values and Map keys are compared using the [SameValueZero][]
-  comparison. (Which means they are free of the [caveats][]).
+1. Primitive values besides NaN are compared using the [Strict Equality
+   Comparison][] ( `===` ). Set and Map values, Map keys and NaN are compared
```

Comment:
We usually use monospacing for constants such as `NaN` in the docs.

Reply:
Addressed